code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -261,6 +261,12 @@ In most cases it makes more sense to use apply_damage() instead! And make sure t
 	var/datum/limb/picked = pick(parts)
 	if(picked.take_damage(brute,burn,sharp,edge))
 		UpdateDamageIcon()
+
+	if(brute)
+		camo_off_process(SCOUT_CLOAK_OFF_DAMAGE, brute) //If we have the Scout cloak, check for a short out
```

Comment:
same thing as above this proc will get called anytime humans take damage.

Reply:
And this.